apps/openmw/mwgui/dialogue.cpp

```diff
@@ -364,6 +364,7 @@ namespace MWGui
         bool sameActor = (mPtr == actor);
         mPtr = actor;
         mTopicsList->setEnabled(true);
+        mTopicsList->scrollToFirstItem();
```

Comment:
As I have written above, in vanilla MW scrollbars are reset when the player press the close button (Ok,  Cancel, Close etc.).